packages/bpk-component-card/src/BpkCardWrapper.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import PropTypes from 'prop-types';
+import { type Node } from 'react';
+
+import { cssModules } from '../../bpk-react-utils';
+
+import STYLES from './BpkCardWrapper.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+type Props = {
+  card: Node,
+  className: ?string,
+  backgroundColor: string,
+  header: Node,
+};
+
+const BpkCardWrapper = (props: Props) => {
+  const { backgroundColor, card, className, header } = props;
+
+  const classNames = getClassName('bpk-card-wrapper', className);
+
+  return (
+    <div
+      className={classNames}
+      style={{
+        '--background-color': backgroundColor,
+      }}
+    >
+      <div
+        style={{ backgroundColor }}
+        className={getClassName('bpk-card-wrapper--header')}
+      >
+        {header}
+      </div>
+      <div className={getClassName('bpk-card-wrapper--content')}>{card}</div>
+    </div>
+  );
+};
+
+BpkCardWrapper.propTypes = {
+  backgroundColor: PropTypes.string.isRequired,
```

Comment:
Wondering if this should not be required but optional and the default colour should be `coreAccentDay`, might be worth us checking with design on this. To see if we would always have this

Reply:
Basically, i follow the [design definition](https://www.figma.com/file/irZ3YBx8vOm16ICkAr7mB3/Backpack?node-id=22572%3A28385&t=S5Vk5LFZTG6ellLC-0) for BPK Card Wrapper API. I' ll check with designer whether it's fine to change to optional and set default `coreAccentDay` colour.

Reply:
Some discussion goes [here](https://skyscanner.slack.com/archives/C04G18ZK2TD/p1676860243721029) in slack.

Reply:
Confirmed with Adam again, we final decided backgroundColor keep required. See [here](https://skyscanner.slack.com/archives/C04G18ZK2TD/p1676950931734259)